zerver/lib/push_notifications.py

```diff
@@ -590,20 +591,27 @@ def get_apns_alert_title(message: Message) -> str:
     # For personal PMs, we just show the sender name.
     return message.sender.full_name
 
-def get_apns_alert_subtitle(message: Message) -> str:
+def get_apns_alert_subtitle(message: Message, user_profile: UserProfile,
+                            mentioned_group: Optional[int]) -> str:
```

Comment:
Just a slight nitpick, but can you make the order of the arguments in this method the same as `get_message_payload_apns`? It's just better for visual consistency.